tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceDialogTest.java

```diff
@@ -392,6 +393,19 @@ public void testReplaceAndFindAfterInitializingFindWithSelectedString() {
 		assertEquals(4, (target.getSelection()).y);
 	}
 
+	@Test
+	public void testIncrementalSearchOnlyActiveWhenAllowed() {
+		openTextViewer("text text text");
+		openFindReplaceDialogForTextViewer();
+
+		dialog.incrementalCheckBox.setSelection(true);
+		dialog.regExCheckBox.setSelection(true);
+		dialog.regExCheckBox.notifyListeners(SWT.Selection, null);
```

Comment:
Please use the existing `select()` method instead, which already does the combination of setting the selection and informing listeners.

Reply:
Done in https://github.com/eclipse-platform/eclipse.platform.ui/commit/d654b06728ceb6865a378e7608c0fec4e1ed13f3